test/python/algorithms/state_fidelities/test_compute_uncompute.py

```diff
@@ -66,7 +81,10 @@ def test_4param_pairs(self):
         fidelity = ComputeUncompute(self._sampler)
         n = len(self._left_params)
         job = fidelity.run(
-            [self._circuit[0]] * n, [self._circuit[1]] * n, self._left_params, self._right_params
```

Comment:
It looks like this was reformatted accidentally -- if you remove the `,` after the last argument (in this case `self._right_params`), black should format it back to what it was before 🙂

Reply:
Thank you!